storybook/src/Twfiek/Atoms/Progress/Progress.js

```diff
@@ -3,7 +3,8 @@ import PropTypes from "prop-types";
 
 const Progress = (props) => {
   const { progress, row = false, disabled = false } = props;
-  let completeClass = progress < 50 ? "error" : progress < 100 ? "primary" : "success";
+  let completeClass =
+    progress < 50 ? "error" : progress < 100 ? "primary" : "success";
```

Comment:
use [classNames](https://github.com/JedWatson/classnames) library. it solves this issue elegently